specification/_types/mapping/RuntimeFields.ts

```diff
@@ -21,7 +21,7 @@ import { Dictionary } from '@spec_utils/Dictionary'
 import { Field } from '@_types/common'
 import { Script } from '@_types/Scripting'
 
-export type RuntimeFields = Dictionary<Field, RuntimeField | RuntimeField[]>
+export type RuntimeFields = Dictionary<Field, RuntimeField>
```

Comment:
Do we know why this originally was made an array?

Reply:
No idea. There's no occurrence of the array variant in our recordings and [looking at the ES source code](https://github.com/elastic/elasticsearch/blob/2a00f25f573e1c1078c1f864eea52397a081c5c8/server/src/main/java/org/elasticsearch/index/mapper/RuntimeField.java#L186) it has always expected the mapping to be an object and not an array.